src/components/ha-expansion-panel.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { fireEvent } from "../common/dom/fire_event";
+import "./ha-svg-icon";
+import { mdiChevronDown } from "@mdi/js";
+import { classMap } from "lit-html/directives/class-map";
+
+@customElement("ha-expansion-panel")
+class HaExpansionPanel extends LitElement {
+  @property({ type: Boolean, reflect: true }) expanded = false;
+
+  @property({ type: Boolean, reflect: true }) outlined = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div class="summary" @click=${this._toggleContainer}>
+        <div class="summary-content">
+          <slot name="title"></slot>
+        </div>
+        <div class="summary-icon ${classMap({ expanded: this.expanded })}">
+          <ha-svg-icon .path=${mdiChevronDown}></ha-svg-icon>
+        </div>
+      </div>
+      <div class="container ${classMap({ expanded: this.expanded })}">
+        <slot name="content"></slot>
+      </div>
+    `;
+  }
+
+  private _toggleContainer(): void {
+    this.expanded = !this.expanded;
+    fireEvent(this, this.expanded ? "expanded" : "collapsed");
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      :host {
+        display: block;
+      }
+
+      :host([outlined]) {
+        box-shadow: none;
+        border-width: 1px;
+        border-style: solid;
+        border-color: var(
+          --ha-card-border-color,
+          var(--divider-color, #e0e0e0)
+        );
+      }
+
+      .summary {
+        display: flex;
+        padding: 0px 16px;
+        min-height: 48px;
+        position: relative;
```

Comment:
Why position relative?

Reply:
Not sure why :) but not needed. I'll remove it.

Reply:
done